coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java

```diff
@@ -121,7 +126,29 @@ public Response<Object> decommission(@PathParam("id") String serverId) {
         });
   }
 
+  @POST
```

Comment:
`GET` can be used instead of `POST` and unit tests added.

Reply:
> So the question is that there should not be a POST request to return all applications?

From a personal perspective, I believe using `POST` requests is better for the following main reasons:

Security: POST requests send data in the request body rather than exposing it in the URL like GET requests. This makes POST requests more secure and helps avoid interception or caching of sensitive information.

Data volume: POST requests have no data length limit and can send large amounts of data to the server. In contrast, GET requests append data to the URL and are subject to URL length limitations.

Reply:
However, in our production environment, `GET` might be sufficient since we have concurrent tasks that do not exceed 200.